extensions/feedbin/package.json

```diff
@@ -47,16 +47,14 @@
   ],
   "commands": [
     {
-      "name": "all-feeds",
+      "name": "view-feeds",
```

Comment:
Can we keep the original name, since people will lose ranking and hotkeys when changing?

Reply:
Ah, sure! Didn't know the ranking was tied to this.

Reply:
It is, it's the identifier for hotkeys, aliases and ranking for the command